src/PrestaShopBundle/DependencyInjection/Compiler/ModulesDoctrineCompilerPass.php

```diff
@@ -85,14 +87,30 @@ private function getCompilerPassList(array $activeModules)
                 }
                 $modulePrefix = 'Module' . Inflector::camelize($moduleFolder->getFilename());
                 $moduleEntityDirectory = realpath($moduleFolder . '/src/Entity');
-                $mappingPass = DoctrineOrmMappingsPass::createAnnotationMappingDriver([$moduleNamespace], [$moduleEntityDirectory], [], false, [$modulePrefix => $moduleNamespace]);
+                $mappingPass = $this->createAnnotationMappingDriver($moduleNamespace, $moduleEntityDirectory, $modulePrefix);
                 $mappingsPassList[$moduleEntityDirectory] = $mappingPass;
             }
         }
 
         return $mappingsPassList;
     }
 
+    /**
+     * @param string $moduleNamespace
+     * @param string $moduleEntityDirectory
+     * @param string $modulePrefix
+     *
+     * @return DoctrineOrmMappingsPass
+     */
+    private function createAnnotationMappingDriver($moduleNamespace, $moduleEntityDirectory, $modulePrefix)
+    {
+        $reader = new Reference('annotation_reader');
+        $driverDefinition = new Definition('Doctrine\ORM\Mapping\Driver\AnnotationDriver', [$reader, [$moduleEntityDirectory]]);
+        $driverDefinition->addMethodCall('addExcludePaths', [[$moduleEntityDirectory . '/index.php']]);
```

Comment:
Consider adding a comment explaining that this whole method exists for the sole purpose of avoiding `exit` found in index files.

Reply:
You're right! Done